packages/cubejs-query-orchestrator/test/unit/QueryOrchestrator.test.js

```diff
@@ -748,8 +748,9 @@ describe('QueryOrchestrator', () => {
     await queryOrchestrator.fetchQuery(internalPreAggregation);
     await queryOrchestrator.fetchQuery(externalPreAggregation);
     await queryOrchestrator.fetchQuery(internalPreAggregation);
+    await queryOrchestrator.fetchQuery(externalPreAggregation);
```

Comment:
I added this to ensure that cache in internal and in external case, works as expected